Tasks/random.js

```diff
@@ -0,0 +1,12 @@
+// Generate random integer value in given range
+
+Random = (min, max) => {
+if ( max === undefined ){;
+    max = min; return Math.floor(Math.random() * (max + 1));
+    ;} else {;
+    return min + Math.floor(Math.random() * (max - min + 1));
+};
```

Comment:
```suggestion
  }
```
Add space before bracket and remove ;